tests/unit/cli/admin/test_import_targets.py

```diff
@@ -0,0 +1,541 @@
+import datetime
+
+import pretend
+import pytest
+
+from repository_service_tuf.cli.admin import import_targets
+
+
+class TestImportTargetsFunctions:
+    def test__check_csv_files(self, monkeypatch):
+        monkeypatch.setattr(
+            import_targets.os.path,
+            "isfile",
+            pretend.call_recorder(lambda *a: True),
+        )
+
+        result = import_targets._check_csv_files(["1of2.csv", "2of2.csv"])
+        assert result is None
+        assert import_targets.os.path.isfile.calls == [
+            pretend.call("1of2.csv"),
+            pretend.call("2of2.csv"),
+        ]
+
+    def test__check_csv_files_file_not_found(self, monkeypatch):
+        monkeypatch.setattr(
+            import_targets.os.path,
+            "isfile",
+            pretend.call_recorder(lambda *a: False),
+        )
+
+        with pytest.raises(import_targets.click.ClickException) as err:
+            import_targets._check_csv_files(["1of2.csv", "2of2.csv"])
+
+        assert "CSV file(s) not found: 1of2.csv, 2of2.csv" in str(err)
+        assert import_targets.os.path.isfile.calls == [
+            pretend.call("1of2.csv"),
+            pretend.call("2of2.csv"),
+        ]
+
+    def test__parse_csv_data(self, monkeypatch):
+        fake_data = [
+            "path/file1;123;blake2b-256;hash1",
+            "path/file2;456;blake2b-256;hash2",
+        ]
+        fake_file_obj = pretend.stub(
+            __enter__=pretend.call_recorder(lambda: fake_data),
+            __exit__=pretend.call_recorder(lambda *a: None),
+            close=pretend.call_recorder(lambda: None),
+            read=pretend.call_recorder(lambda: fake_data),
+        )
```

Comment:
I think is the same what @MVrachev [mentioned above](https://github.com/vmware/repository-service-tuf-cli/pull/195#discussion_r1104658018)

Actually, `get_headers` returns the headers used for specific calls (other endpoints) after checking if the token is still valid (`is_logged`) and returns the headers formatted.

We can have another helper in the `api_client` to retrieve the bootstrap status. Because the status will be used in different behaviors.

Reply:
@MVrachev , @KAUTH 
I was simulating this change here, it will endup in a lot of changes.
I think would be better do it in a different PR, otherwise it will deviate from the initial PR goal.